Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -61,7 +61,12 @@ class FontDescription {
     FontWidthVariant widthVariant() const { return static_cast<FontWidthVariant>(m_widthVariant); }
     const FontFeatureSettings& featureSettings() const { return m_featureSettings; }
     const FontVariationSettings& variationSettings() const { return m_variationSettings; }
-    FontSynthesis fontSynthesis() const { return static_cast<FontSynthesis>(m_fontSynthesis); }
+    FontSynthesisLonghandValue fontSynthesisWeight() const { return static_cast<FontSynthesisLonghandValue>(m_fontSynthesisWeight); }
+    FontSynthesisLonghandValue fontSynthesisStyle() const { return static_cast<FontSynthesisLonghandValue>(m_fontSynthesisStyle); }
+    FontSynthesisLonghandValue fontSynthesisSmallCaps() const { return static_cast<FontSynthesisLonghandValue>(m_fontSynthesisCaps); }
+    bool isFontSynthesisWeightEnabled() const { return fontSynthesisWeight() == FontSynthesisLonghandValue::Auto; }
+    bool isFontSynthesisStyleEnabled() const { return fontSynthesisStyle() == FontSynthesisLonghandValue::Auto; }
+    bool isFontSynthesisSmallCapsEnabled() const { return fontSynthesisSmallCaps() == FontSynthesisLonghandValue::Auto; }
```

Comment:
`Enabled` feels like the wrong word here (Enabled typically refers to some feature setting)

maybe `hasAutoFontSynthesisWeight`?